src/screens/playing.rs

```diff
@@ -16,13 +17,39 @@ pub(super) fn plugin(app: &mut App) {
     );
 }
 
-fn spawn_level(mut commands: Commands) {
+#[derive(Resource, Asset, Reflect, Clone)]
+pub struct GameplayMusic {
```

Comment:
I don't like the duplication of structures here. This really feels like it should be a single resource synced to the game state. Only one track should be playing at once.

Reply:
Resolving this as I think we've agreed that distributed is the move here.